system/helpers.php

```diff
@@ -116,3 +116,41 @@ function d($var = false, $to_file = false)
         }
     }
 }
+
+/**
+ * Application container
+ */
+if (! function_exists('app')) {
+    function app()
+    {
+       return App::getContainer();
+    }
+}
+
+/**
+ * Path to root directory
+ *
+ * @return string
+ */
+if (! function_exists('root_path')) {
+    function root_path(string $path = ''): string
+    {
+        $path = preg_replace('/(\/+)/', DS, $path);
+
+        return dirname(__DIR__) . DS.$path.DS;
+    }
+}
+
+/**
+ * Path to assets directory
+ *
+ * @return string
+ */
+if (! function_exists('assets_path')) {
```

Comment:
We already have the constant [ASSETS_PATH](https://github.com/simba77/johncms/blob/develop/system/bootstrap.php#L41), why also a helper?  
He's not needed